v1.0/coordinatePoint.schema.json

```diff
@@ -1,22 +1,22 @@
 {
   "_type": "https://openminds.ebrains.eu/sands/CoordinatePoint",
-  "description": "Structured information describing a coordinate point in a space",
+  "description": "Structured information describing a coordinate point in a defined space.",
   "required": [
     "coordinates",
     "coordinateSpace"
   ],
   "properties": {
     "coordinates": {
       "type": "array",
-      "description": "Coordinates that describe the coordinate point (x, y , z).",
+      "_instruction": "Enter the coordinates (x, y , z; 2D, with z = 0) that describe this coordinate point, defined in the unit of the coordinate space.",
       "items": {
         "type": "number"
```

Comment:
I don't see why not. The instructions just need to make it clear that this can't be a min/maxValue but values (+ unit). Curators would need to check for unit compatability (coordinate unit vs. coordinateSpace unit). If no units are stated, I guess we assume that it's in the unit of the coordinate space?

Reply:
Follow-up question: Do we have any way to define required field when schemas are referenced in other schemas, so that we can define individual required fields for different references? 

For example, SCHEMA1 has "_linkedType" to "quantitativeValue.schema.json" and would require value and unit. SCHEMA2 has same "_linkedType" but would require only a value. "quantitativeValue.schema.json" doesn't have any required fields. 

I guess that's not possible, right?